services/rook-ceph-cluster/1.11.7/pre-install.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   wait: true
```

Comment:
we'll also need need to bump this Kustomizations name

https://github.com/mesosphere/kommander-applications/blob/main/services/rook-ceph-cluster/1.11.4/pre-install.yaml#L4

and https://github.com/mesosphere/kommander-applications/blob/main/services/rook-ceph-cluster/1.11.4/rook-ceph-cluster.yaml#L9

Reply:
cc @bcmendoza (if this can be made part of automation)